perf-tool/include/perf.hpp

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef __PERF_H__
+#define __PERF_H__
+
+#include <json.hpp>
+#include <sys/types.h>
+#include <unistd.h>
+#include <string>
```

Comment:
Looks like string is already included in `json.hpp`

Reply:
I have a different opinion on this one: we should include anything that the file refers directly and not rely on internals of other hpp files. If in the future we no longer need json.hpp in this file, or if json.hpp is changed such that no longer includes <string> then we'll have a build failure.

Reply:
That makes sense